src/container/Wrapper.js

```diff
@@ -21,14 +20,18 @@ export class Wrapper extends Component {
     navigation: PropTypes.object.isRequired
   };
 
-
   render() {
     const { audio } = this.props;
 
     return (
       <View style={{ flex: 1 }}>
         {this.props.children}
-        <AudioPlayer data={{ chapters: audio.chapters, selected_chapter: audio.selected_chapter }} />
+        <AudioPlayer
+          data={{
+            chapters: audio.chapters,
+            selected_chapter: audio.selected_chapter
```

Comment:
we need to set a rule if we'll follow the underscore based or camel case based name convention for name variables